src/index.js

```diff
@@ -117,12 +113,14 @@ export const getTargets = (targets = {}) => {
 
     delete targetOps.electron;
   }
+  browserslist.defaults = objectToBrowserslist(targetOps);
+
+  const browsersQuery = targetOps.browsers;
+  const browserslistOpts = {path: opts.path};
```

Comment:
Ahh, gotcha.

I guess my only real nit with the name "path" is that in a list of options I'd find myself asking "path to _what_?"

How about `configPath`? I dunno. Ultimately, it's not a big issue and am fine with it if @hzoo or anyone else doesn't care :)